test/test_distributions.py

```diff
@@ -2169,6 +2178,101 @@ def test_lazy_probs_initialization(self):
                 self.assertFalse('logits' in vars(dist), msg=message)
 
 
+@unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+class TestAgainstScipy(TestCase):
+    def setUp(self):
+        set_rng_seed(0)  # see Note [Randomized statistical tests]
```

Comment:
nit: I believe the note is unnecessary here: this test should always pass; it does not have a known positive failure rate. You should even be able to remove the `set_rng_seed(0)` line, since that happens automatically for `TestCase` subclasses IIRC.

Reply:
I had play it a little bit for `LogNormal`, because when `scale`>4, variance gets very large so tests fails for 1e-5 tolerance. Now that I'm clamping it manually, it should be safe to remove.